lib/oembed/utils.py

```diff
@@ -7,12 +7,16 @@
 from htk.lib.oembed.constants import *
 from htk.utils.request import get_current_request
 
-def get_oembed_html(url):
+def get_oembed_html(url, autoplay=False):
     """Gets the oEmbed HTML for a URL, if it is an oEmbed type
     """
     oembed_type = get_oembed_type(url)
+
     if oembed_type:
-        html = get_oembed_html_for_service(url, oembed_type)
+        if oembed_type == "youtube":
+            html = youtube_oembed(url, autoplay)
```

Comment:
this works, but for style/readability/documentation/prevent human errors, we should do named arguments explicitly instead of implicitly by position:

`autoplay=autoplay`

Reply:
so you want a line that explicitly states `autoplay = autoplay`?